File: kfc.py

# Sample input
# dinein        takeaway
# 2               3
# Pizza          Coke
# Coke           Burger
#                Pasta
# Sample output
# Ordered items: Pizza,Coke
# Total amount: 300.0

# Ordered items: Coke,Burger,Pasta
# Total amount: 300

a=input()
b=int(input())                             
arr=[]
total=0
for i in range(b):
    v=list(map(str,input().split()))
    arr+=v

for i in arr:
    if i=="Pizza":
        total+=200
    elif i=="Pasta":
        total+=80
    elif i=="Burger":
        total+=150
    elif i=="Coke":
        total+=50
if a=="dinein":
    temp=total*20/100
    total+=temp
else:
    total+=20
res=""
for i in range(b):
    res+=arr[i]
    if i!=b-1:
      res+=","
print("Ordered items:",res)
print("Total amount:",total)

File: kfc.java
import java.util.*;
class A{
    public static void main(String [] args){
        Scanner in=new Scanner(System.in);
        int total=0;
        
        String a=in.nextLine();
        int b=in.nextInt();
        in.nextLine();  
        String res="";
        for(int i=0;i<=b;i++){
            String v1=in.next();
            res+=v1;
            if(i!=b-1){
                res+=",";
            }
        }
        String arr[]=res.split(",");
        for(String v:arr){
            if(v.equals("Pizza")){
                total+=200;
            }
            else if(v.equals("Burger")){
                total+=150;

            }
             else if(v.equals("Coke")){
                total+=50;

            }
             else if(v.equals("Pasta")){
                total+=100;

            }}
    if(a.equals("dinein")){
        int temp=total*10/100;

        total+=temp;
    }
    else{
        total+=10;
    }
        System.out.println("Ordered items:"+res);
        System.out.println("Total Bill:"+total);
        }
    }
